.gitignore

```
@@ -0,0 +1 @@
+*.class
```

You should put some exception below in .gitignore
```suggestion
*.o
*.out
*.exe 
```

---

Thanks for your feedback. I'll work on it